vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -185,12 +252,29 @@ void learn_or_predict(ccb& data, multi_learner& base, multi_ex& examples)
   sanity_checks<is_learn>(data);
   create_cb_labels(data);
 
+  std::vector<std::string> generated_interactions;
```

Comment:
The perf of this is iffy. It's fine to use as-is for now. Are we tracking perf work that we'll need to do later?

Reply:
I agree it is iffy, I don't have a tracking item for CCB perf yet.